typescript-data-model/order-history.ts

```diff
@@ -0,0 +1,78 @@
+interface OrderDetails {
+  orderNumber: string;
+  totalCost: number;
+  destination: string;
+  dateOrderPlaced: string;
+  deliveryDate: string;
+  items: {
+    itemName: string;
+    seller: string;
+    itemCost: number;
+  }[];
```

Comment:
For something like this I would recommend making a second interface for an Item and then making `items: Item[]` instead of creating the object type inside of another interface. It makes it more readable and easier to understand/break down.

Reply:
Ya that makes more sense. Thank you!